pkg/asset/installconfig/alibabacloud/alibabacloud.go

```diff
@@ -17,6 +17,13 @@ const (
 	defaultAcceptLanguage = "en-US"
 )
 
+var unsupportedRegions = map[string]bool{
```

Comment:
Can you use `sets` from here: https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#String

Here is an example using sets in this package: https://github.com/openshift/installer/blob/master/pkg/types/azure/validation/machinepool.go#L21-L24

Reply:
Thanks for your suggestion, I have updated.